sdk/build.gradle

```diff
@@ -48,4 +54,33 @@ task bumpAndTag {
     doLast {
         LibraryProject.INSTANCE.tagAndIncrement(LibraryProject.INSTANCE.getLibraryVersionCode() + 1)
     }
+}
+
+tasks.withType(Javadoc).all {
+    /* Disable javadoc since we do not have any java file */
+    enabled = false
+}
+
+// Because the components are created only during the afterEvaluate phase, you must
+// configure your publications using the afterEvaluate() lifecycle method.
+afterEvaluate {
+    publishing {
+        publications {
+            // Creates a Maven publication called "release".
+            release(MavenPublication) {
+                // Applies the component for the release build variant.
+                from components.release
+
+                // You can then customize attributes of the publication as shown below.
+                version = LibraryProject.INSTANCE.getLibraryVersionName()
+            }
+            // Creates a Maven publication called “debug”.
+            debug(MavenPublication) {
```

Comment:
I would just skip the debug publication, I don't really see a use case and it pollutes the task list.